libmscore/layout.cpp

```diff
@@ -2543,6 +2543,58 @@ void layoutDrumsetChord(Chord* c, const Drumset* drumset, const StaffType* st, q
             }
       }
 
+//---------------------------------------------------------
+//   extendedStemLenWithTwoNotesTremolo
+//    Extend stem of one of the chords to make the tremolo less steep
+//---------------------------------------------------------
+
+std::pair<qreal, qreal> extendedStemLenWithTwoNoteTremolo(Tremolo* _tremolo, qreal stemLen1, qreal stemLen2)
```

Comment:
Could you add comments that contain values?
`std::pair<qreal/*...*/, qreal/*...*/> `

Reply:
I'm about to add a comment but in the comment section above. This line is long enough :-)